resources/js/components/ui/button.tsx

```diff
@@ -18,7 +18,7 @@ const buttonVariants = cva(
         secondary:
           "bg-zinc-100 text-zinc-900 shadow-sm hover:bg-zinc-100/80 dark:bg-zinc-800 dark:text-zinc-50 dark:hover:bg-zinc-800/80",
         ghost: "hover:bg-zinc-100 hover:text-zinc-900 dark:hover:bg-zinc-800 dark:hover:text-zinc-50",
-        link: "text-zinc-900 underline-offset-4 hover:underline dark:text-zinc-50",
```

Comment:
We don't need to change this as shadcn already has a button that looks like this: https://ui.shadcn.com/docs/components/button#ghost. So on the `Navbar` change "Login" and "Register" to have variant `ghost` and only make the corners sharp on those buttons instead of all of them.

Reply:
I reverted the changes I made to the link variant. I made a few changes to the ghost variant to make the style more consistent for dark mode